lib/giraph/schema.rb

```diff
@@ -0,0 +1,46 @@
+module Giraph
+  # GraphQL::Schema wrapper allowing decleration of
+  # resolver objects per op type (query or mutation)
+  class Schema < GraphQL::Schema
```

Comment:
Subclassing classes that you don't control is always a little dangerous, since changes to them can unexpectedly break your implementation.